docs/UserGuide.md

```diff
@@ -69,23 +69,24 @@ Want to use MESS immediately? Let's get started!
  
 #### Adding a task or event: `add`
 Adds a task to the to-do list<br>
-Format: `add TASK_NAME [starts START_DATETIME ends CLOSE_DATETIME tag TAG recurs NUMBER_OF_WEEKLY_RECURRING_TASK]`
+Format: `add TASK_NAME [starts START_DATETIME ends CLOSE_DATETIME tag TAG]`
 
 > Date format of START_DATE and CLOSE_DATE includes words like today, tomorrow, 3 days from now, day after tomorrow, noon, 12pm, 6am
 
+> Use ISO date-time format for typing exact date. e.g. 2016/9/1 represents 1st September 2016
```

Comment:
Change to

Use ISO date-time format if you wish to type exact date. e.g. 2016/9/1 represents 1st September 2016